db/migrate/20211203171247_migrate_linkedin_user_info.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class MigrateLinkedinUserInfo < ActiveRecord::Migration[6.1]
```

Comment:
Is there going to be a future migration that removes the old data?

Reply:
Yup, I'm planning to add one next week, after we're 100% sure the move was successful

(and likewise for the `discourse-microsoft-auth` plugin I migrated yesterday)